core/src/components/item/test/alignment/index.html

```diff
@@ -27,7 +27,7 @@
           <ion-list-header>Leading Icons</ion-list-header>
           <ion-item>
             <ion-icon slot="start" name="time"></ion-icon>
-            <ion-datetime display-format="DDDD MMMM D YYYY hh:mm:ss a" value="2019-10-01T15:43:40.394Z"></ion-datetime>
+            <ion-datetime value="2019-10-01T15:43:40.394Z"></ion-datetime>
```

Comment:
Do we need to use `ion-datetime` here? Wondering if we can use something else instead to make the screenshots a bit smaller

Reply:
I don't believe so, I can swap those out with inputs 👍